x/inflation/handler.go

```diff
@@ -0,0 +1,24 @@
+package inflation
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/errors"
```

Comment:
Nit: use `errortypes` alias to be consistent with codebase
```suggestion
	errortypes "github.com/cosmos/cosmos-sdk/types/errors"
```